workflow/qa-steps.md

```diff
@@ -5,6 +5,17 @@ description: Recommendations for common QA steps when reviewing PRs
 
 In this section you will find a list of steps to take when reviewing PRs, in order to catch unforeseen regressions.
 
+## General guidelines for quality reviews
+
+- Assume positive intent, we all have collective responsibility for the code we write - be nice.
+- Avoid expressing subjective opinions without reasoning. If you think something is bad, you need to explain _why_ you think it is bad.
```

Comment:
Does _underscore_ do the same thing as *star*? I didn't know that :smile:

Reply:
It does - double underscore will also result in __bold__. No idea why they are two ways of doing it. 🤷🏻‍♂️